packages/system/src/components/Formatters/Date.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { DateFormatterOptions, useDateFormatter } from '@react-aria/i18n';
+
+interface DateFormatterProps extends DateFormatterOptions {
+  value: Date;
+  fullDigitWidth?: boolean;
+}
+
+export const DateFormatter = ({
+  value,
+  fullDigitWidth,
```

Comment:
Not sure if devs will understand that name. I think "full" is kind of misleading?

Reply:
hmm, ok i have some suggestions for this,
for example : wideDigits, tabularDigits, expandedDigits, numericWidth
@sebald